revenger/infrastructure/python_adapter_factory.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from infrastructure.generic_classes import GenericLogger, GenericPythonAdapter, GenericSaver
+from infrastructure.python_adapter import PythonAdapter
+from infrastructure.python_adapter_v2 import PythonAdapterV2
+
+from os import environ
+class PythonAdapterFactory:
+    PY_ADAPTER_LATEST_ENV_VAR = "PY_ADAPTER_LATEST"
```

Comment:
The design is very good. The point is that we do no need anymore the old quick and dirty implementation I had done, or do you see any reason to still keep it?